styles.css

```diff
@@ -1 +1,74 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+/*Grid mobile*/
+@media screen and (max-width: 800px) {
```

Comment:
Kun je weglaten. Dit is je standaard scherm beschrijving. De andere media gaat pas werken bij een minimale breedte van wat je hebt ingevuld.